Exiled.Loader/ConfigManager.cs

```diff
@@ -114,12 +114,78 @@ public static class ConfigManager
             }
         }
 
+        /// <summary>
+        /// Loads all plugin translations.
+        /// </summary>
+        /// <param name="rawTranslations">The raw translations to be loaded.</param>
+        /// <returns>Returns a dictionary of loaded translations.</returns>
+        public static SortedDictionary<string, ITranslations> LoadSortedTranslations(string rawTranslations)
```

Comment:
The `Sorted` was added because of backward compatibility to the name of the config load method. No requirement to do the same with that method.